### PR TITLE
Update DomainTools.xml

### PR DESCRIPTION
**Pull request by koops76**
#11269

### CODE REVIEW

**Comment by J0WI** [comment]
Dupe of #11270

**Comment by koops76** [comment]
@J0WI I guess domaintools.com blocks Travis IP range so I will add it to the whitelist.

**Comment by koops76** [comment]
@J0WI Done.

File: src/chrome/content/rules/DomainTools.xml [hunk 1]
```diff
@@ -1,95 +1,71 @@
...
+	<target host="secure.domaintools.com" />
+	<target host="source.domaintools.com" />
+	<target host="thumbnails.domaintools.com" />
+	<target host="whitelabel.domaintools.com" />
+	<target host="whois.domaintools.com" />
+	<target host="yahoo.domaintools.com" />
+	
+	<target host="assets.dt-static.com" />
+	<target host="images.dt-static.com" />
+	<target host="img.dt-static.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/DomainTools.xml]: Does not exisit

**Comment by koops76** [comment]
@J0WI Done.

**Comment by J0WI** [comment]
Please also update the hash in the whitelist.

**Comment by wesdawg** [comment]
I've added the whitelist in #11270 
This can be closed.

**Comment by koops76** [comment]
@J0WI Updated the whitelist, merge please.

**Pull request merged**